### DISCUSSION TITLE
Where to find `run.go` file

### DISCUSSION THREAD

**Discussion by aerfio** [golangci/golangci-lint#3017] [Q&A] [answered]
Docs sometimes mention usage of `run.go` file, like here https://golangci-lint.run/usage/linters/#gofumpt
```
 # Deprecated: use the global `run.go` instead.
```

but otherwise docs dont mention this file anywhere, and I couldnt find anything about it in this repo.

I suspect that it's just meaning that those linters will take go version from the go.mod file, am I wrong?

**Comment by ldez** [comment, marked as answer]
Hello,

it's not a file but a configuration:
```yml
run:
   go: xxx
```

https://golangci-lint.run/usage/configuration/#run-configuration

**Comment by aerfio** [comment]
🤦🏻 oh god yeah, it's obvious 😅 Thanks you!
